src/SourceBuild/tarball/content/ci.yml

```diff
@@ -0,0 +1,19 @@
+variables:
```

Comment:
Does this pipeline need a PR trigger?

Reply:
For PR validation on Azure Repos Git, it's configured via the UI and not yaml: https://learn.microsoft.com/en-us/azure/devops/pipelines/repos/azure-repos-git?view=azure-devops&tabs=yaml#pr-triggers

I'll put a `trigger: none` here for now, but do you think we should have it trigger on commits to release/* branches as well as for PR validation?